lib/index.js

```diff
@@ -71,8 +60,19 @@ Klaviyo.prototype.loaded = function() {
 
 Klaviyo.prototype.identify = function(identify) {
   // TODO: should map/alias the rest of the reserved props
+  var traitAliases = {
+    email: '$email',
+    firstName: '$first_name',
+    lastName: '$last_name',
+    phone: '$phone_number',
+    title: '$title'
+  };
+  // don't add $id if email is enforced
```

Comment:
@hankim813 Can you clarify these comments a bit more? I'm having trouble understanding `enforceEmail` and what this does